sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -2464,6 +2388,82 @@
                   "addons": [],
                   "id": "eqekqf"
                 },
+                {
+                  "label": "HTML",
+                  "attrs": [
+                    {
+                      "attr": "",
+                      "value": ""
+                    }
+                  ],
+                  "content": "Identify schedule breaks for the study period offering:\n<ul>\n  <li>\n    Study breaks of five or fewer consecutive calendar days that would normally be instructional days do not count towards study breaks.\n  </li>\n  <li>Study breaks cannot exceed three weeks, or 21 consecutive days. Excess break days will be deducted from overall funded weeks.</li>\n  <li>\n    Breaks in study may not exceed 10% (individually or cumulatively) of the study period offering length. Excess break days will be deducted from overall funded weeks.\n  </li>\n</ul>",
+                  "refreshOnChange": false,
+                  "customClass": "align-bullets mt-n2",

```

Comment:
for the info, why do we need `mt-n2`

Reply:
The customClass is not part of the PR. It was only content change. Having said that, I believe we used mt-n2 reduce the vertical space between header and sub header. 

With mt-n2
![image](https://github.com/bcgov/SIMS/assets/54600590/b08c1702-18bc-4cca-a8e9-a82aa2026cde)

Without mt-n2

![image](https://github.com/bcgov/SIMS/assets/54600590/88f2c841-daed-4f26-8e9e-3e02f6c6821a)

Reply:
Thanks for the comment, I noticed something got flipped on the order and fixed it while investigating the comment.